### ISSUE TITLE
Can't resize Breakpoints section in Debug sidebar

### ISSUE THREAD

**Issue by gushuro** [microsoft/vscode#46886]
Steps to reproduce:
1. Open debug sidebar 
2. Try to resize the last section `Breakpoints` using the mouse.

If I swap the section with another, everyone works fine. It seems to be specific to `Breakpoints` when it's in the bottom.

I see the same behaviour in 1.21 and today's 1.22-insiders.

**Comment by isidorn** [comment]
That is by design of the breakpoints view since our first version.
It has fixed size and is docked to the bottom by default.

The downside of this is that currently breakpoints view is the only view that behaves like this

No other user complained about this so I am relucatant to change it. For now let's leave it as a feature request and only change if we get more feedback about this.

**Comment by hitesh-parmar** [comment]
I still face this issue. It is not possible to resize breakpoints view. I have many breakpoints and it is frustrating to scroll and look for a breakpoint you want to deactivate. I interchanged the position of the breakpoints view but still no luck. 

Any reason why it is the way it is? I feel the behaviour should be same for all the panes.
@isidorn Could you please have a look again and see if this can be planned.

**Comment by isidorn** [comment]
Sure we can reopen this. Though still not planned.

**Comment by weinand** [comment]
@isidorn
What exactly was the reason that the breakpoints view cannot be resized?
Is there any reason why removing the constraint would be a problem?

**Comment by isidorn** [comment]
@weinand it just feels better since it does not require manual size managment by the user.
Like the open editors view.

**Comment by rushvora** [comment]
@isidorn Open editors section can be resized when the scroll bar is present. Breakpoints cannot be resized regardless of the number of breakpoints present.

**Comment by weinand** [comment]
@rushvora good point
@isidorn could we make "Breakpoints" view and "Open editors" view behave the same?

**Comment by isidorn** [comment]
Makes sense. I have pushed a change that the Breakpoint behave like the Open Editors.

**Comment by isidorn** [comment]
Since I see a lot of thumbs up and no complaints I am adding a verified label.
Users please let me know if this is not correct and this feature does not work for you.

**Comment by rushvora** [comment]
@isidorn Tested in the latest insider build - Version: 1.45.0-insider. No extensions installed, fresh build I just downloaded to try this out. (I mistakenly thought the feature hadn't made it to stable since it's still not draggable)

It partly works. It's still not draggable, but the size does expand, only when all the other sections are collapsed. I have attached a screenshot to show the same, where the breakpoint section expands to occupy all the space.

<img width="267" alt="Screenshot 2020-04-28 at 05 51 15" src="https://user-images.githubusercontent.com/602333/80433436-8d3a9780-8914-11ea-834e-84f191372d82.png">

In the below screenshot, when any one or more section(s) are expanded, the size is still fixed. Dragging the breakpoints upward expands the call stack section, and I can't seem to increase the size of the breakpoint section in this scenario.

<img width="259" alt="Screenshot 2020-04-28 at 05 53 58" src="https://user-images.githubusercontent.com/602333/80433541-d8ed4100-8914-11ea-8e9b-062cf1b98afe.png">

So as of now the only way to see an expanded breakpoints section is the collapse all the other sections (variables, watch, call stack).

Hope this feedback helps!
